packages/store/src/lib/flags.ts

```diff
@@ -15,12 +15,13 @@ export const storeFlags = {
 
 export const fileFlags = {
   'from-file': Flags.string({
-    description: 'The SQLite file to import data from.',
+    description: "The SQLite file to import data from. Can't be used with --to-file.",
```

Comment:
Extreme nit for preposition order while we're updating text.

```suggestion
    description: "The SQLite file from which to import data. Can't be used with --to-file.",
```